fcrepo-kernel-modeshape/src/test/java/org/fcrepo/kernel/modeshape/services/VersionServiceImplTest.java

```diff
@@ -37,6 +37,8 @@
 import org.fcrepo.kernel.api.exception.RepositoryRuntimeException;
 import org.fcrepo.kernel.api.services.VersionService;
 import org.fcrepo.kernel.modeshape.FedoraSessionImpl;
+
+import org.junit.Assert;
```

Comment:
That's what I do, but some people do just `extend Assert`, which I don't think is wrong. The important thing is to tighten it up.